frontend/src/components/Footer.tsx

```diff
@@ -84,7 +84,7 @@ const Footer = () => {
         <Grid row gap>
           <Grid tablet={{ col: 4 }} desktop={{ col: 6 }}>
             <Image
-              className="maxh-15 margin-bottom-2 tablet:margin-bottom-0"
+              className="margin-bottom-2 tablet:margin-bottom-0"
```

Comment:
Why was `. maxh-15` removed? It's there so that the logo doesn't grow too large on mobile. Larger breakpoints add a max size. But on mobile the image will just fill the allotted space and be HUGE. 

![image](https://github.com/navapbc/simpler-grants-gov/assets/409279/dadb77b1-6802-456d-ab81-5d1ce4ade14a)

Reply:
The max width creates an error in the console:

```
warn-once.js:16 Image with src "/_next/static/media/grants-gov-logo.7f89963e.png" has either width or height modified, but not the other. If you use CSS to change the size of your image, also include the styles 'width: "auto"' or 'height: "auto"' to maintain the aspect ratio
```

I created a separate ticket for it since it is out of scope for this ticket: https://github.com/navapbc/simpler-grants-gov/issues/165